src/startupscriptgenerator/src/common/parseUserCommands.go

```diff
@@ -0,0 +1,61 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ParseUserRunCommand(sourcePath string) string {
+
+	appsvcFile, err := os.Open(sourcePath)
+	if err != nil {
```

Comment:
As mentioned in a previous discussion, if the user doesn't have an `appsvc.yaml` file, then we shouldn't display any errors as they shouldn't care about a file they didn't provide not being found.